content/BatchResponseContent.md

```diff
@@ -10,13 +10,15 @@ See [constraints](ContentArchitecturalConstraints.md) related to all content obj
 
 - Deserialize batch response payload into a collection of GraphResponse objects
 - Allow users to enumerate collection of GraphResponse objects
-- Expose NextLink URL if one is present in the payload.
```

Comment:
The nextLink would be part of a response, right? Why wouldn't we expose it in this case?

Reply:
Removing this will address the documentation update via https://github.com/microsoftgraph/microsoft-graph-docs/pull/11934

The spec before implied that the batch response would have a nextLink property at the root of the response payload. This isn't really the case, and the nextLink of a collection response can already be obtained by deserializing the individual response item to its collection object as you would any other object.

Reply:
Yeah, that seems awfully wrong. Yeah, agree to remove.